ddl/ddl_api.go

```diff
@@ -2589,7 +2589,16 @@ func (d *ddl) AlterTable(ctx context.Context, sctx sessionctx.Context, ident ast
 		case ast.AlterTableDropStatistics:
 			err = d.AlterTableDropStatistics(sctx, ident, spec.Statistics, spec.IfExists)
 		case ast.AlterTableAttributes:
-			err = d.AlterTableAttributes(sctx, ident, spec)
+			var val string
+			val, err = variable.GetGlobalSystemVar(sctx.GetSessionVars(), variable.TiDBEnableAlterAttributes)
```

Comment:
why not use `EnableAlterAttributes` in `SessionVars` struct directly?

Reply:
Ah.. you reminds me that, this **global** variable does not need **sessionvars** at all. It is a historical behavior to cache global variable in the session.